distributed/client.py

```diff
@@ -2571,6 +2571,7 @@ def __anext__(self):
             raise StopAsyncIteration  # flake8: noqa
         while self.queue.empty():
             yield self.event.wait()
+        sleep(0.0000000001)
```

Comment:
No.  You might be able to get something from [one of the failed tracebacks](https://ci.appveyor.com/project/daskdev/distributed/build/1.0.871) however